STEERING-COMMITTEE.md

```diff
@@ -60,28 +59,27 @@ committee currently recognizes this delegated authority for:
 
 KSC meets every two weeks, or as-needed. Meetings are held online.
 
-Given the private nature of many of these discussions (e.g. privacy, private
+Given the private nature of some of these discussions (e.g. privacy, private
 emails to the committee, code of conduct violations, escalations, disputes
-between members, security reports, etc.) meetings are held in private.
+between members, security reports, etc.) some meetings are held in private.
```

Comment:
I would like to see a policy that states that by default all meetings will be publicly available, that exceptions would be only when meeting to discuss privacy/private matters. Is ok to have held meetings and split the time between public and private portion, the private portion will not be recorded.

Reply:
I don't disagree but I am going to refrain from incorporating that for now in order to keep the focus on the most important parts. Hope it's okay to revisit that later on.

Reply:
I'm going to resolve this for now; we can come back to it once the key parts are resolved.